fmlcore/src/main/java/net/minecraftforge/fml/config/ConfigFileTypeHandler.java

```diff
@@ -67,7 +68,9 @@ public void unload(Path configBasePath, ModConfig config) {
     }
 
     private boolean setupConfigFile(final ModConfig modConfig, final Path file, final ConfigFormat<?> conf) throws IOException {
-        Files.createDirectories(file.getParent());
+        try {
+            Files.createDirectories(file.getParent());
```

Comment:
Interesting, not a fan of how they resolved the linked JDK issue.
However, in this case I think it would be better to guard this creates call with a !Files.exists like we usually do in File based io operations. Makes it more clear that we're not using exceptions as code flow.